class-gwiz-gf-code-chest.php

```diff
@@ -208,16 +208,13 @@ public function init() {
 
 		load_plugin_textdomain( $this->_slug, false, basename( dirname( __file__ ) ) . '/languages/' );
 
-		if ( current_user_can( 'administrator' ) ) {
-			add_filter( 'gform_tooltips', array( $this, 'tooltips' ) );
-			add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_editor_script' ) );
-			add_action( 'gform_post_save_feed_settings', array( $this, 'save_code_chest_settings' ), 10, 4 );
-			add_filter( 'gform_noconflict_scripts', array( $this, 'noconflict_scripts' ) );
-			add_filter( 'gform_noconflict_styles', array( $this, 'noconflict_styles' ) );
-
-			// 11 so that this comes right after and can override the legacy Custom JS plugin setting config.
-			add_filter( 'gform_form_settings_fields', array( $this, 'replace_custom_js_setting' ), 11, 2 );
-		}
+		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_editor_script' ) );
+		add_action( 'gform_post_save_feed_settings', array( $this, 'save_code_chest_settings' ), 10, 4 );
+		add_filter( 'gform_noconflict_scripts', array( $this, 'noconflict_scripts' ) );
+		add_filter( 'gform_noconflict_styles', array( $this, 'noconflict_styles' ) );
+
+		// 11 so that this comes right after and can override the legacy Custom JS plugin setting config.
+		add_filter( 'gform_form_settings_fields', array( $this, 'replace_custom_js_setting' ), 11, 2 );
```

Comment:
Thoughts on moving all of this into the `init_admin` method? That should make it a bit more compartmentalized and keep from registering all of this stuff on the frontend.